src/snowflake/snowpark/modin/plugin/extensions/groupby_overrides.py

```diff
@@ -1427,7 +1388,10 @@ def unique(self):
 
     def size(self):
         # TODO: Remove this once SNOW-1478924 is fixed
```

Comment:
is the linked issue resolved by this?

Reply:
Oh - great point! I'll tag this issue in this PR as well!